test/aievec/conv2d_i8_after_polygeist.mlir

```diff
@@ -38,7 +38,7 @@ module attributes {dlti.dl_spec = #dlti.dl_spec<#dlti.dl_entry<"dlti.endianness"
 //      CHECK:    %[[C16:.*]] = arith.constant 16 : index
 //      CHECK:    %[[C0:.*]] = arith.constant 0 : index
 //      CHECK:    %[[T0:.*]] = aievec.upd %[[A1]][%[[C0:.*]]] {index = 0 : i8, offset = 0 : i32} : memref<?xi8>, vector<64xi8>
-//      CHECK:    %[[T1:.*]] = aievec.shuffle %[[T0:.*]] {mode = 0 : i32} : vector<64xi8>, vector<64xi8>
+//      CHECK:    %[[T1:.*]] = aievec.legacyshuffle %[[T0:.*]] {mode = 0 : i32} : vector<64xi8>, vector<64xi8>
```

Comment:
Given the mode=0 (`shuffle_T8_64x2_lo`), I'm a bit confused why this can work without a second operand...

Reply:
"Shuffle" always takes two arguments. When translating to C++, the single operand "shuffle" C++ intrinsic calls a two operand version with "undef" in the rhs. For the new `shuffle`, we will have to inject the `undef` when lowering to `LLVM IR`. With the new constraints, I'm just trying to keep things semantically consistent.